src/components/StyleGuide/Typography.mdx

```diff
@@ -130,9 +527,9 @@ used for body copy, captions and secondary captions.
   borderColor="ui.border.default"
   borderRadius="5px"
   mb="m"
-  p="s"
+  p="inset.default"
 >
-  <Text>
+  <Text fontFamily="body" fontSize="1rem" m="0">
```

Comment:
Storybook is overriding styles that are naturally coming through some of the components.  I had to do this to ensure that the correct styles would show when the component was rendered.  But only because site users will not see the actual code.